department-of-veterans-affairs/caseflow
# Adding id to issues and making issue keys unique in SideBarDocumentInformation, ClaimsFolderDetails

connects #3533

Test plan:
- Open the Case folder page in reader for a specific case that has issues
- Make sure all the issues show up in the Case folder details accordion
- Open a document, make sure all the issues show up in the sidebar in the case details section
- Open the brown console and make sure flattenChildren - Encountered two children with the same key warning message doesn't show up.
- Also make sure rest of reader works as expected
